src/simplewallet/simplewallet.cpp

```diff
@@ -3623,6 +3637,28 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
   if (local_args.size() > 0 && parse_priority(local_args[0], priority))
     local_args.erase(local_args.begin());
 
+  if (priority == 0 && m_wallet->get_default_priority() != 1 && m_wallet->try_lowest_priority())
+  {
+    try
+    {
+      double fee_level = m_wallet->get_fee_multiplier(1) * m_wallet->get_per_kb_fee() / (double)1024;
+      std::vector<std::pair<uint64_t, uint64_t>> blocks = m_wallet->estimate_backlog({std::make_pair(fee_level, fee_level)});
+      if (blocks.size() != 1)
+      {
+        fail_msg_writer() << tr("Error: bad estimated backlog array size");
+      }
+      else if (blocks[0].first == 0)
+      {
+        success_msg_writer() << tr("There's no backlog for the lowest priority fee level, so using the lowest priority.");
```

Comment:
Minor, but these really should be using the normal writer.

Reply:
ok